docs/source/basic/cheatsheet.rst

```diff
@@ -109,7 +109,11 @@ Allocate a buffer in host memory
   .. code-block:: c++
 
      // Use alpaka vector as a static array for the extents
-     Vec<Dim, Idx> extent = value;
+     // If Dim is DimInt<1>;
+     Vec<Dim, Idx> extent = scalarIdxValue;
```

Comment:
to me this would be more clear as
```suggestion
     Vec<Dim, Idx> extent = value;
```
or
```suggestion
     Vec<DimInt<1>, Idx> extent = value;
```